server/game/gameSystems/PlayCardSystem.ts

```diff
@@ -0,0 +1,81 @@
+import type { Card } from '../core/card/Card';
+import AbilityResolver from '../core/gameSteps/AbilityResolver';
+import { CardTargetSystem, ICardTargetSystemProperties } from '../core/gameSystem/CardTargetSystem';
+import { AbilityContext } from '../core/ability/AbilityContext';
+import * as Contract from '../core/utils/Contract';
+import { CardType, PlayType } from '../core/Constants';
+import { isPlayable } from '../core/card/CardTypes';
+import * as GameSystemLibrary from './GameSystemLibrary';
+import { PlayCardAction } from '../core/ability/PlayCardAction';
+import { PlayUnitAction } from '../actions/PlayUnitAction';
+import { PlayUpgradeAction } from '../actions/PlayUpgradeAction';
+import { PlayEventAction } from '../actions/PlayEventAction';
+
+export interface IPlayCardProperties extends ICardTargetSystemProperties {
+    ignoredRequirements?: string[];
+
```

Comment:
One other additional feature I just thought of - we'll need to eventually work out a way to flag whether this action should be nested or not. E.g., for the Vader ability it's not nested but for U-Wing it is. That can be a TODO if you want since it may get kind of tricky, but we should add some comments at least (probably here and on the method in GameSystemLibrary).